docs/install/python_wheels.md

```diff
@@ -210,7 +210,7 @@ On BB5, we can test CPU wheels with:
 ```
 salloc -A proj16  -N 1 --ntasks-per-node=4 -C "cpu" --time=1:00:00 -p interactive
 module load unstable python
-bash packaging/python/test_wheels.sh python3.8 wheelhouse/NEURON-7.8.0.236-cp38-cp38m-manylinux1_x86_64.whl
+bash packaging/python/test_wheels.sh python3.9 wheelhouse/NEURON-7.8.0.236-cp39-cp39m-manylinux1_x86_64.whl
```

Comment:
Slight nitpick: should be `cp39-cp39` (without the `m` suffix) since from 3.8 onwards the ABI flags [are empty](https://docs.python.org/3/library/sys.html#sys.abiflags).